syntax = "proto3";

enum MessageType {
  REQ_LOGIN = 0;
  RES_LOGIN = 1;
}
